src/components/FeeSummary/package-lock.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
It turns out we probably don't need these package lock files after all.

Oddly, "lerna hoist" created these files once when I ran it. But then I opened up a new window and ran everything from scratch, they weren't created (and everything still worked).

So, I've deleted all these now :)